_commands/basics/seq.md

```diff
@@ -0,0 +1,75 @@
+---
+---
+
+seq
+-------
+
+`seq`prints a sequence of numbers to standard ouput.
+
+~~~ bash
+$ seq [options]... [*FIRST* [*STEP*]] *LAST*... 
+~~~
+
+<!--more-->
+
+where `seq` prints numbers from *FIRST* to *LAST* by an increment *STEP*.
+Default: *FIRST* & *STEP* are both 1. Each number is printed on its own line. Seq can be used on all real numbers.
+
+### Useful Options / Examples
+
+#### `-f, --format=`*FORMAT*
+
+prints all numbers using *FORMAT*, where the default is `%g`. *FORMAT* must contain one of the following float output formats: `%g`, `%e`, `%f`. 
```

Comment:
I'm a little confused, you say that FORMAT has to have one of %g, %e, or %f, but your example below this doesn't have any of those?